packages/common/src/utils/wallet.test.ts

```diff
@@ -0,0 +1,146 @@
+import { AUDIO, wAUDIO, USDC, FixedDecimal } from '@audius/fixed-decimal'
+import BN from 'bn.js'
+
+import { BNWei, BNUSDC } from 'models/Wallet'
+
+import {
+  parseAudioInputToWei,
+  formatWei,
+  convertWAudioToWei,
+  convertWeiToWAudio,
+  ceilingBNUSDCToNearestCent,
+  floorBNUSDCToNearestCent,
+  formatUSDCWeiToUSDString,
+  formatUSDCWeiToCeilingDollarNumber,
+  formatUSDCWeiToCeilingCentsNumber,
+  formatUSDCWeiToFloorDollarNumber,
+  formatUSDCWeiToFloorCentsNumber,
+  convertBigIntToAmountObject
+} from './wallet'
+
+describe('wallet.ts currency formatting and conversion tests', function () {
+  it('can parse audio input to wei', function () {
+    const audioInput = '12345'
+    const expected = '12345000000000000000000'
+    expect(AUDIO(audioInput).value.toString()).toBe(expected)
```

Comment:
note these tests are also parity tests for the new `FixedDecimal` as well as tests for the old helper methods. Look to this file for recipes!